### PR TITLE
Portfolio - detail popup window

### PR DESCRIPTION
**Pull request by Strangeal**
### Detail Popup Window

- Added javascript to render cards section dynamically
- Added popup window for mobile screens
- Added popup window for wide screens

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -408,6 +428,110 @@ textarea {
...
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.disable {
+  display: none;
+}
+
```
Comment (sja-thedude) [inline styles.css]: - [x] Kindly make sure that the background of the popup should be blurred and transparent as per the attached loom video.
- [x] Kindly make sure to display a light gray colour horizontal line after tech stack in the mobile popup as per the attached loom video.
- [x] Kindly make sure to centre buttons in the mobile popup as per the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Strangeal, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ All linters are passing

## Required 
- [ ] Kindly view this loom video in order to make changes [here](https://www.loom.com/share/5bfbff20bac748799910649dc9731338).
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -408,6 +428,115 @@ textarea {
...
 
+/* PopUp Window */
+.feature {
+  background-color: #fff;
+  width: 60%;
+  height: 95%;
+  padding: 0 2rem;
+  overflow-y: scroll;
+  border-radius: 16px;
+}
```
Comment (leonardodiasb) [inline styles.css:439]: - [x] Your popup modal is great and you did a great job implementing the suggested changes. But the text in the buttons on the mobile version is not aligned after your changes. I suggest you play around with the padding, width, and font-size of your modal and elements inside so they can look more like the template.

 [OPTIONAL] Your modal cannot display the entire content on the screen and the user have to scroll down to look for the buttons. This is not the best user experience and would be nicer if everything in the modal is possible to see right after the modal is opened. Try playing around with the dimension of your modal(width), reducing the padding on the sides, changing the font-sizes of the elements inside, and making sure the values match with the values from the Figma template.
 -Your mobile modal:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Strangeal, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

✔️ Professional Readme
✔️ Correct Gitflow
✔️ All linters are passing

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Strangeal 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**